sdk/ai/ai-inference-rest/test/public/utils/recordedClient.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  Recorder,
+  RecorderStartOptions,
+  VitestTestContext,
+  assertEnvironmentVariable,
+} from "@azure-tools/test-recorder";
+import { AzureKeyCredential } from "@azure/core-auth";
+import { ClientOptions } from "@azure-rest/core-client";
+import createClient, { ModelClient } from "../../../src/index.js";
+
+const envSetupForPlayback: Record<string, string> = {
+  AZURE_ENDPOINT: "https://endpoint",
+  AZURE_CLIENT_ID: "azure_client_id",
+  AZURE_CLIENT_SECRET: "azure_client_secret",
+  AZURE_TENANT_ID: "88888888-8888-8888-8888-888888888888",
```

Comment:
These are redacted by default, no need to specifying them.

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-js/pull/29863/commits/a7695376837b7478b4978de5baf1a83d43f264ab)